docs/retrieve.txt

```diff
@@ -0,0 +1,342 @@
+      F1 world championship three times before his death at age 34.
+
+.. important:: Required File Changes
+
+   To run the preceding code example, you must edit the ``MovieController.php`` file in your
+   ``my-app`` application. Copy the code example and paste it into the ``show()`` function, replacing
+   any existing code inside this function.
+
+Retrieve All Documents
+~~~~~~~~~~~~~~~~~~~~~~
+
+You can perform a find operation without specifying a query filter, which retrieves all
+documents in a collection. Use the following syntax to run a find operation that matches all
+documents:
+
+.. code-block:: php
+
+   $movies = Movie::get();
+
+Alternatively, use the ``get()`` method's alias ``all()`` to retrieve all documents.
+
+.. warning::
+
+   The ``movies`` collection in the Atlas sample dataset stores a large amount of data.
+   Retrieving and displaying all documents in this collection may cause your web 
+   application to time out.
+
+Retrieve One Matching Document
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve only one document that matches a set of criteria, use the ``where()`` method
+followed by the ``first()`` method. By default, the ``first()`` method returns the first
+matching document according to the documents' natural order, or as they appear in the
+database. If you apply a sort to the find operation, ``first()`` returns the first matching
+document according to the sorted order. 
+
+To learn more about sorting, see the following resources:
+
+- :manual:`Natural order </reference/glossary/#std-term-natural-order>`
+  in the Server manual glossary
+- `Ordering, Grouping, Limit and Offset
+  <https://laravel.com/docs/10.x/queries#ordering-grouping-limit-and-offset>`__ in the Laravel
+  documentation
+   
+The following example returns one document in which the value of the ``title`` field
+is ``"The Parent Trap"``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: php
+      
+      return view('browse_movies', [
+         'movies' => Movie::where('title', 'The Parent Trap')
+            ->first()
+      ]);
+
+   .. output::
+      :language: none
+      :visible: false
+
+      Title: The Parent Trap
+      Year: 1961
+      Runtime: 129
+      IMDB Rating: 7.1
+      IMDB Votes: 12768
+      Plot: Teenage twin girls swap places and scheme to reunite their divorced parents.
+
+Without chaining the ``first()`` method to the ``where()`` method, the preceding example returns
+two documents in which the ``title`` value is ``"The Parent Trap"``.
+
+.. important:: Required File Changes
+
+   To run the preceding code example, you must edit the ``MovieController.php`` file in your
+   ``my-app`` application. Copy the code example and paste it into the ``show()`` function, replacing
+   any existing code inside this function.
+
+Use Laravel Queries to Retrieve Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Rather than on a model, you can run find operations on a query builder instance that
```

Comment:
S: the first clause is a bit confusing
```suggestion
You can run find operations on a query builder instance that represents your collection instead of running them on models.
```